javaclient/src/main/java/confluo/streaming/ConfluoProducer.java

```diff
@@ -0,0 +1,157 @@
+package confluo.streaming;
+
+import confluo.rpc.*;
+import confluo.streaming.config.ProducerConfig;
+import java.io.Closeable;
+import org.apache.thrift.TApplicationException;
+import org.apache.thrift.TException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Properties;
+
+/**
+ * ConfluoProducer will create the topic if it isn't existed,or reuse the old topic. batch produce
+ * message can be enable if batch size was set to bigger than 1,not thread safe
+ */
+public class ConfluoProducer implements Closeable {
+  Logger logger = LoggerFactory.getLogger(ConfluoProducer.class);
+  private String topic;
+  private String host;
+  private int port;
+  private long totalProducedNum;
+  private int produceBatchSize;
+  private boolean batchEnable;
+  private int messageSize;
+  private rpc_storage_mode storageMode;
+  private String produceSchema;
+  private RpcClient client;
+  private MessageBatchBuilder batchBuilder;
+  private Schema curSchema;
+  private int batched;
+
+  public ConfluoProducer(Properties properties) throws TException {
+    host =
+        properties.getProperty(
+            ProducerConfig.BOOTSTRAP_ADDRESS, ProducerConfig.BOOTSTRAP_ADDRESS_DEFAULT);
```

Comment:
Any specific reason for calling this `BOOTSTRAP_ADDRESS` and `BOOTSTRAP_PORT`? I think it would be clearer to call it `HOST_ADDRESS` and `HOST_PORT`.

Reply:
ok